app/models/currency_conversion.rb

```diff
@@ -4,12 +4,24 @@
 #
 # Table name: currency_conversions
 #
-#  rate          :decimal
+#  id            :bigint           not null, primary key
 #  from_currency :string
+#  rate          :decimal(, )
 #  to_currency   :string
-#  conference_id :integer
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  conference_id :bigint           not null
+#
+# Indexes
+#
+#  index_currency_conversions_on_conference_id  (conference_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (conference_id => conferences.id)
 #
 class CurrencyConversion < ApplicationRecord
+  VALID_CURRENCIES = %w[AUD CAD CHF CNY EUR GBP JPY USD].freeze
   belongs_to :conference
```

Comment:
Are these the only currencies that will be available? If not, it may be more convenient to use a library to not hardcode all the currencies available

Reply:
ignore comment, clarification was done